### ISSUE TITLE
How to Disable the Day list

### ISSUE THREAD

**Issue by vc-akashkadia** [flippingbitss/react-daterange-picker#16]
![image](https://user-images.githubusercontent.com/49644157/93190872-7f6d9d80-f761-11ea-95f2-49e0e455dc04.png)

How to remove this list

**Comment by rbandooni** [comment]
@vc-akashkadia I don’t think this has been documented by the author, however, after looking at the source code I figured out the following:

Add `definedRanges={[]}` as attribute to the component.

e.g. 
``` jsx
<DateRangePicker
definedRanges={[]}
open={dateRangeOpen}
onChange={(range) => setDateRange(range)}
  />
```
